docs/source/cli_help/traverse.rst

```diff
@@ -13,10 +13,10 @@ traverse
 
 .. code-block:: shell
 
-    git machete traverse [-F|--fetch] [-l|--list-commits] [-M|--merge]
-                         [-n|--no-edit-merge|--no-interactive-rebase] [--no-detect-squash-merges]
-                         [--[no-]push] [--[no-]push-untracked]
-                         [--return-to=WHERE] [--start-from=WHERE] [-w|--whole] [-W] [-y|--yes]
+    git machete t[raverse] [-F|--fetch] [-l|--list-commits] [-M|--merge]
```

Comment:
Also! since this change affects the command-line interface (and not just the underlying implementation), pls bump the version to 3.6.0 (plus also update release notes)